docs/open-payments.md

```diff
@@ -4,4 +4,8 @@ Open Payments is an API standard that allows third-parties (with the account hol
 
 Rafiki implements the Open Payments APIs. Every request and response is validated against the [Open Payments API specification](https://github.com/interledger/open-payments/tree/main/openapi).
 
+The specification version that Rafiki uses is configured in [fetch-schemas.sh](../packages/open-payments/scripts/fetch-schemas.sh). Currently, it is using the version with commit hash [146ff684cc003149fb7362861a3b24b40dddb31c](https://github.com/interledger/open-payments/tree/146ff684cc003149fb7362861a3b24b40dddb31c/openapi).
```

Comment:
I'm not sure we should specify the commit hash here since I expect `fetch-schemas.sh` will be updated without updating these docs.

Reply:
I think we should specify the tag once we have tagged it. Hence the TODO. And we do have the "documentation added" checklist item, that we probably should take more serious.